Add October 2024 newsletter (#2239)
writethedocs/www: docs/blog/newsletter-october-2024.rst

:og:image: https://www.writethedocs.org/_static/logo-opengraph.png

.. post:: October 3, 2022
  :tags: newsletter

########################################
Write the Docs Newsletter – October 2024
########################################

> Hello hello, wonderful documentarians! I hope you all are able to find sources of inspiration around you. The ideas coming from the Write the Docs community and conferences recently have provided me with hope for the future that I hope continues to inspire.

Reviewer: Double hope is odd?
```suggestion
Hello hello, wonderful documentarians! I hope you all are able to find sources of inspiration around you. The ideas coming from the Write the Docs community and conferences recently have provided me with hope for the future.
```

Author: Yeah, that did feel weird to me even when writing it.

> The most recent conference was the Atlantic 2024 conference. If you missed it, or just want to recall the insights, check out the `videos from the conference <https://www.youtube.com/@writethedocs>`__. And once you've seen those, check out the next conference, Australia 2024, which just announced its `speakers and full schedule </conf/australia/2024/news/announcing-speakers-schedule/>`__. Be sure to grab a ticket for November 28–29 in Melbourne if you can.

Author: ```suggestion
The most recent conference was the Atlantic 2024 conference. If you missed it, or just want to recall the insights, check out the `videos from the conference <https://www.youtube.com/playlist?list=PLZAeFn6dfHpn8IckCiREggN0f9oWEMomW>`__. And once you've seen those, check out the next conference, Australia 2024, which just announced its `speakers and full schedule </conf/australia/2024/news/announcing-speakers-schedule/>`__. Be sure to grab a ticket for November 28–29 in Melbourne if you can.
```

> The most recent conference was the Atlantic 2024 conference. If you missed it, or just want to recall the insights, check out the `videos from the conference <https://www.youtube.com/playlist?list=PLZAeFn6dfHpn8IckCiREggN0f9oWEMomW>`__. And once you've seen those, check out the next conference, Australia 2024, which just announced its `speakers and full schedule </conf/australia/2024/news/announcing-speakers-schedule/>`__. Be sure to grab a ticket for November 28–29 in Melbourne if you can.

Reviewer: ```suggestion
Our most recent conference was the Atlantic 2024 conference. If you missed it, or just want to revisit, check out the `videos from the conference <https://www.youtube.com/playlist?list=PLZAeFn6dfHpn8IckCiREggN0f9oWEMomW>`__. And once you've seen those, check out the next conference, Australia 2024, which just announced its `speakers and full schedule </conf/australia/2024/news/announcing-speakers-schedule/>`__. Be sure to grab a ticket for November 28–29 in Melbourne if you can.
```

Reviewer: Could mention sketchnotes here as well?

Author: ```suggestion
Our most recent conference was the Atlantic 2024 conference. If you missed it, or just want to revisit, check out the `videos from the conference <https://www.youtube.com/playlist?list=PLZAeFn6dfHpn8IckCiREggN0f9oWEMomW>`__ and the `sketchnotes of every talk <https://www.flickr.com/photos/writethedocs/albums/72177720320644083>`__. And once you've seen those, check out the next conference, Australia 2024, which just announced its `speakers and full schedule </conf/australia/2024/news/announcing-speakers-schedule/>`__. Be sure to grab a ticket for November 28–29 in Melbourne if you can.
```

> In other community news, the 2024 WTD Documentation Salary Survey - our 6th annual survey - will be open for submissions very soon! We're implementing a couple of frequently-requested usability improvements and are still ironing out a few issues, but keep an eye on the `survey website <https://salary-survey.writethedocs.org/>`__ and follow Write the Docs on Twitter and LinkedIn, where it will be announced as soon as it's ready.

Author: ```suggestion
In other community news, the 2024 WTD Documentation Salary Survey -- our 6th annual survey -- will be open for submissions very soon! We're implementing a couple of frequently requested usability improvements and are still ironing out a few issues, but keep an eye on the `survey website <https://salary-survey.writethedocs.org/>`__ and follow Write the Docs on Twitter and LinkedIn, where it will be announced as soon as it's ready.
```

> In other community news, the 2024 WTD Documentation Salary Survey - our 6th annual survey - will be open for submissions very soon! We're implementing a couple of frequently-requested usability improvements and are still ironing out a few issues, but keep an eye on the `survey website <https://salary-survey.writethedocs.org/>`__ and follow Write the Docs on Twitter and LinkedIn, where it will be announced as soon as it's ready.

Author: ```suggestion
In other community news, the 2024 WTD Documentation Salary Survey -- our 6th annual survey -- will be open for submissions very soon! We're implementing a couple of frequently-requested usability improvements and are still ironing out a few issues, but keep an eye on the `survey website <https://salary-survey.writethedocs.org/>`__ and follow Write the Docs on Twitter and LinkedIn, where it will be announced as soon as it's ready.
```

Relatedly, the Write the Docs job board has been replaced. In its place, we have a `new sponsorship opportunity </sponsorship/jobs/>`__ for for companies looking to hire documentarians. We're still hoping to provide useful information about job openings, just closer to where our community is.

In the newsletter this month, we have articles on dealing with feedback in large volumes, how to handle freelance work, standards for using callouts, and whether to keep docs in code. Hope you'll find something for you!

------------------------------
Managing mountains of feedback
------------------------------

Resolving feedback on documentation can mean anything from making straightforward updates to undertaking a substantial investigation and revision, but it always takes time away from documenting new product functionality. If you can't hire more documentarians to keep up with both addressing feedback and writing new docs, you can try these ideas.

* Tag feedback tickets with information about their required effort so that writers can take on the quick fixes as a mental break from more involved work. Schedule bigger tickets and critical issues alongside product work.

> * If it works with your release cadence, schedule time each month or quarter for a bug bash devoted entirely to addressing docs feedback.

Author: ```suggestion
* If it works with your release cadence, schedule time each month or quarter for a "bug bash" devoted entirely to addressing docs feedback.
```

> * If it works with your release cadence, schedule time each month or quarter for a bug bash devoted entirely to addressing docs feedback.

Author: ```suggestion
* If it works with your release cadence, schedule time each month or quarter for a "bug bash" devoted entirely to addressing docs feedback.
```

Some documentarians consider contracting or freelancing while job hunting. In the `#freelance channel <https://writethedocs.slack.com/archives/CA1J9GV17>`__, a former employee asked about taking on a client.

**What rate to use?**

Rates for technical writers vary widely and depend on many factors. 

* The domain and the audience: In software, developer documentation prompts the highest rates; end-user documentation rates are usually lower. Hardware-associated documentation rates may be lower. Specialist documentation (such as scientific or medical) may command higher rates. 
* The geographical location: In the US, tech hubs, major cities, and the coasts have higher rates. In non-tech industries and non-coastal areas, rates tend to be lower. Rates vary by country and relate to cost of living in a particular area. 
* The type of work requested: Charge less for minor editing. Rewriting or creating new content costs more. Re-organizing content (information architecture) or, in a docs-as-code environment, setting up and working with a tool chain would command higher rates.
* Your skills and experience: Highly skilled and experienced professionals can ask for higher rates.

> To calculate an hourly rate (in the US) determine the salary for an equivalent position, divide that by 52x40, and then double or triple that to compensate for benefits. Check job postings and the  `2023 WTD Salary Survey <https://www.writethedocs.org/surveys/salary-survey/2023/>`_ for salaries. Freelancers and consultants can ask for higher rates than contractors.

Reviewer: > Freelancers and consultants can ask for higher rates than contractors.

this feels odd to me, especially without an internationally agreed upon definition, consider deleting

Author: ```suggestion
To calculate an hourly rate (in the US) determine the salary for an equivalent position, divide that by 52x40, and then double or triple that to compensate for benefits. Check job postings and the  `2023 WTD Salary Survey <https://www.writethedocs.org/surveys/salary-survey/2023/>`_ for salaries.
```

**Can I assume that the client pays taxes?**

> If you’re working as a W2 contractor in the US, the agency takes out federal taxes. As a freelancer (1099), you pay all taxes. In other countries, check your local tax requirements, which vary widely.

Reviewer: ```suggestion
If you’re working as a W2 contractor in the US, the agency takes out federal taxes. As a freelancer in the US (1099), you pay all taxes. In other countries, check your local tax requirements, which vary widely.
```

**How often will I get paid?**

As a contractor, the pay period depends on the agency. As a freelancer, your contract defines the pay period. Some freelancers bill based on the project (milestones); some calculate invoices based on an hourly rate. 

**What can I consider billable hours?**

As a contractor working on site, you normally bill for hours on site based on your contract. As a freelancer working off site, you bill whenever you work on the project. In general, you don't get overtime unless it’s in your contract.

---------------------------------------
Using callouts/admonitions in tech docs
---------------------------------------

> When it comes to creating clear and effective technical documentation, knowing when to use callouts and admonitions (boxes that stand out from the surrounding text) and avoiding overuse is key to creating user-friendly content. They can highlight important information and ensure your documentation remains accessible and easy to navigate.

Author: ```suggestion
When it comes to creating clear and effective technical documentation, knowing when to use callouts and admonitions (sections, often boxes, that stand out from the surrounding text) and avoiding overuse is key to creating user-friendly content. They can highlight important information and ensure your documentation remains accessible and easy to navigate.
```

Some style guides follow a tiered system to ensure proper use:

- Tip: Helpful advice that can make tasks easier but isn’t critical.
- Note: Relevant but non-essential information that adds context but doesn’t affect the task’s outcome.
- Important: Information that, if missed, could lead to frustration or additional work.
- Warning: Critical details about data loss or security risks, where overlooking the warning could cause significant problems.

> The key is moderation. Too many boxed notes, tips, or warnings can cause “admonition fatigue” and lead to readers skipping over important information. When deciding between boxed content or inline text, weigh the impact. If it’s crucial and might be missed, box it. Otherwise, integrating it into the main body ensures smoother flow and readability.

Author: ```suggestion
The key is moderation. Too many individual notes, tips, or warnings can cause “admonition fatigue” and lead to readers skipping over important information. When deciding between separated content or inline text, weigh the impact. If it’s crucial and might be missed, make it stand out. Otherwise, integrating it into the main body ensures smoother flow and readability.
```

By setting clear guidelines and using these tools purposefully, you can help users focus on what’s most important—without overwhelming them.

-------------------------------------
Should APIs be documented with code?
-------------------------------------

A recent question about documenting APIs in code in the `#documenting-apis channel <https://writethedocs.slack.com/archives/C0YH9K2JY>`__ met with some strong pushback. Someone felt that putting documentation in code (meaning the code used to generate the product) was the last thing documentarians should want. Others defended the practice.

The reasons given for separating docs and code included the idea that because docs in code relies on automated tools, their syntax is much less human-readable. It was also noted that modifying code might require specialized skills and privileges that some documentarians don't have. Having to compile and test an entire code source for a small docs change was also viewed as a barrier to entry, whereas a separate reference file means documentarians can work independently. 

> The responses focused on the benefits of automatic generation and how it meant the reference was much more likely to stay up-to-date, especially compared to maintaining API definitions by hand. They celebrated the dependence on the code. It was also seen as a way to meet developers where they already are and so lower their barriers to contributing. People advocating for docs in code also said they keep all formatting in code to a minimum, with anything more complex in a guide separate from the API reference.

Author: ```suggestion
The responses focused on the benefits of automatic generation and how it meant the reference was much more likely to stay up to date, especially compared to maintaining API definitions by hand. They celebrated the dependence on the code. It was also seen as a way to meet developers where they already are and so lower their barriers to contributing. People advocating for docs in code also said they keep all formatting in code to a minimum, with anything more complex in a guide separate from the API reference.
```
